components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/RecoveryScenarios.java

```diff
@@ -44,7 +44,8 @@ public static RecoveryScenarios getRecoveryScenario(String scenarioName) throws
 
         RecoveryScenarios[] scenarios = {
                 NOTIFICATION_BASED_PW_RECOVERY, QUESTION_BASED_PWD_RECOVERY, USERNAME_RECOVERY, SELF_SIGN_UP,
-                ASK_PASSWORD, ADMIN_FORCED_PASSWORD_RESET_VIA_EMAIL_LINK, ADMIN_FORCED_PASSWORD_RESET_VIA_OTP
+                ASK_PASSWORD, ADMIN_FORCED_PASSWORD_RESET_VIA_EMAIL_LINK, ADMIN_FORCED_PASSWORD_RESET_VIA_OTP,
```

Comment:
On a separate note, is this something allowed to registered dynamically or configured outside?

Reply:
No, we are defining here and using it in APIs. Not dynamic.